File: mapir-android-service-sdk/src/main/java/ir/map/servicesdk/enums/FilterOptions.java
package ir.map.servicesdk.enums;

public enum FilterOptions {
    PROVINCE("province"),
    CITY("city"),
    COUNTY("county"),
    REGION("region"),
    NEIGHBOURHOOD("neighborhood"),
    DISTANCE("distance"),
    POLYGON("polygon");

    private String value;

    FilterOptions(String value) {
        this.value = value;
    }

    @Override
    public String toString() {
        return value;
    }
}

File: mapir-android-service-sdk-ktx/src/main/java/ir/map/servicesdk/enums/FilterOptions.kt
package ir.map.servicesdk.enums

enum class FilterOptions(private val value: String) {
    PROVINCE("province"),
    CITY("city"),
    COUNTY("county"),
    REGION("region"),
    NEIGHBOURHOOD("neighbourhood"),
    DISTANCE("distance"),
    POLYGON("polygon");

    override fun toString() = value
}